aleph-client/src/vesting.rs

```diff
@@ -0,0 +1,123 @@
+use anyhow::Result as AnyResult;
+use log::info;
+pub use pallet_vesting::VestingInfo;
+use sp_core::Pair;
+use substrate_api_client::{compose_extrinsic, GenericAddress, XtStatus::Finalized};
+use thiserror::Error;
+
+use primitives::Balance;
+
+use crate::{account_from_keypair, try_send_xt, AccountId, BlockNumber, Connection, KeyPair};
+
+/// Gathers errors from this module.
+#[derive(Debug, Error)]
+pub enum VestingError {
+    #[error("🦺❌ Account has no active vesting schedules.")]
+    NotVesting,
+}
+
+pub type VestingSchedule = VestingInfo<Balance, BlockNumber>;
+
+const PALLET: &str = "Vesting";
+
+/// Calls `pallet_vesting::vest` for `who`, i.e. makes all unlocked balances transferable.
+///
+/// Does not expect `connection` to be signed. Fails if transaction could not have been sent.
+///
+/// *Note*: This function returns `Ok(_)` even if the account has no active vesting schedules
+/// and thus the extrinsic was not successful. However, semantically it is still correct.
+pub fn vest(connection: &Connection, who: KeyPair) -> AnyResult<()> {
+    // Ensure that we make a call as `account`.
+    let connection = connection.clone().set_signer(who.clone());
+    let xt = compose_extrinsic!(connection, "Vesting", "vest");
```

Comment:
You could be using `PALLET` here instead of `"Vesting"` (and below).

Reply:
sure, I must have missed that - fixed